tests/App/KernelTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace App;
+
+use Mmi\App\App;
+use Mmi\Http\Request;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Mmi\IndexController
+ */
+final class KernelTest extends TestCase
+{
+    public function testDefaultResponseIsReturnedForEmptyRequest(): void
+    {
+        $app = new App(new Request());
```

Comment:
Przydałaby się koncepcja "środowisk" w aplikacji, teraz to wczyta wszystkie configi jak leci :sweat_smile:

Reply:
No i brakuje jakiejś "dystrybuowanej" z frameworkiem bazowej klasy do testowania w opartych na nim projektach (wzorem KernelTestCase w Symfony), pomyślę jak to można dorobić.